setup.py

```diff
@@ -8,7 +9,7 @@
     print("=" * 80)
     raise e from None
 
-with open("README.md") as f:
+with open(os.path.join(os.path.dirname(__file__), "README.md")) as f:
```

Comment:
Please use `open` with `encoding='utf-8'` (or `io.open`, if you wish to support Python 2) to prevent the following crash during the installation:
https://github.com/RGF-team/rgf/issues/65
https://github.com/henry0312/pytest-codestyle/pull/34